File: M3Pact.BusinessModel/Admin/M3UserModel.cs
using M3Pact.BusinessModel.BusinessModels;
using System.Collections.Generic;

namespace M3Pact.BusinessModel.Admin
{
    public class M3UserModel
    {
        public UserLoginDTO User { get; set; }
        public string BusinessUnit { get; set; }
        public string Title { get; set; }
        public string ReportsTo { get; set; }
        public string Site { get; set; }
        public List<string> Clients { get; set; }
        public bool IsUserExist { get; set; }
        public bool IsActiveEmployee { get; set; }
    }
}

File: M3Pact.Web/src/app/admin/users/m3user/m3-user.model.ts
import { SelectedClients } from '../all-users/models/selected-clients.model';
import { UserLogin } from '../../../shared/models/user-login';
import { ValidationResponseViewModel } from '../../../common/models/validation.model';


export class M3UserModel extends ValidationResponseViewModel {
    public user: UserLogin;
    public isUserExist: boolean;
    public selectedRole: string;
    public businessUnit: string;
    public title: string;
    public reportsTo: string;
    public Site: string;
    public clients: SelectedClients[];
    public isActiveEmployee: boolean;
    constructor() {
        super();
        this.user = new UserLogin();
        this.clients = new Array<SelectedClients>();
    }
}
